webapp/src/controllers/session.ts

```diff
@@ -159,6 +163,14 @@ export async function publicSessionMessagesData(req, res, _next) {
  * get session messages
  */
 export async function sessionMessagesJson(req, res, next) {
+	if (req?.query?.messageId) {
```

Comment:
Add a chainValidations that takes req.query and checks messageId is 24 hex characters, so it can be converted to object id or this can be an error.

e.g. in addAgentApi:
```
...
	let validationError = chainValidations(
		req.body,
		[
			{ field: 'name', validation: { notEmpty: true, ofType: 'string' } },
			{ field: 'modelId', validation: { notEmpty: true, hasLength: 24, ofType: 'string' } },

...
```

but change that `req.body` -> `req.query`, just have one validation for messageId and then return dynamicresponse if you get an error back, just like in add agent api :slightly_smiling_face: